integration/helper.go

```diff
@@ -72,3 +77,46 @@ func (db Database) Close(ctx context.Context) error {
 func (db Database) URL() string {
 	return db.url
 }
+
+func (db Database) Connect(ctx context.Context) (*pgx.Conn, error) {
+	return pgx.Connect(ctx, db.URL())
+}
+
+func (db Database) Refresh(ctx context.Context) error {
+	con, err := db.Connect(ctx)
+	if err != nil {
+		return fmt.Errorf("couldn't connect to testing database: %w", err)
```

Comment:
As a general rule, I tend to avoid using words like "failed" or "couldn't" in error messages to keep them consider, otherwise when they are all appended, you end up just repeating the word "couldnt" over and over.

I'd prefer "connecting to test database: %w"